fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/service/ChargeEnumerations.java

```diff
@@ -170,7 +170,7 @@ public static EnumOptionData chargeCalculationType(final ChargeCalculationType t
             break;
             case PERCENT_OF_UNUTILIZED_AMOUNT:
                 optionData = new EnumOptionData(ChargeCalculationType.PERCENT_OF_UNUTILIZED_AMOUNT.getValue().longValue(),
-                        ChargeCalculationType.PERCENT_OF_UNUTILIZED_AMOUNT.getCode(), "% Unutilized Amount");
+                        ChargeCalculationType.PERCENT_OF_UNUTILIZED_AMOUNT.getCode(), "% annually of Unutilized Amount");
```

Comment:
Maybe "Unutilized Amount (Annual %)"

Reply:
starting with "%" is a fineract convention,
then should I change all enum types?

Reply:
okay so maybe "% Daily Unutilized Amount (Annual %)". This is quite confusing but it is actually what's going on here...